group1/comm/tests.py

```diff
@@ -1,3 +1,34 @@
 from django.test import TestCase
+from comm.models import Message, Room
+from datetime import datetime
+from django.contrib.auth.models import User
 
 # Create your tests here.
+class RoomTestCase(TestCase):
+    def setUp(self):
+    	#Create room and user objects
+        Room.objects.create(name = "test1", description = "test1")
+        room1 = Room.objects.get(name = "test1")
+        Room.objects.create(name = "test2", description = "test2")
+        room2 = Room.objects.get(name = "test2")
```

Comment:
Hey Will! I just did this because I was concerned that I was deleting columns from the entire messages table to keep it from exceeding 5000, when what I wanted to do was delete messages that belong to a room that has more than 5000. Maybe I can make it more clear by testing to see if the messages table itself has 10000 messages, like it should.